src/main/java/com/datapipe/jenkins/vault/credentials/common/VaultStringCredentialBinding.java

```diff
@@ -0,0 +1,52 @@
+package com.datapipe.jenkins.vault.credentials.common;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import java.io.IOException;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.credentialsbinding.Binding;
+import org.jenkinsci.plugins.credentialsbinding.BindingDescriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class VaultStringCredentialBinding extends Binding<VaultStringCredential> {
+
+
+    @DataBoundConstructor
+    public VaultStringCredentialBinding(String variable, String credentialsId) {
+        super(variable, credentialsId);
+    }
+
+    @Override protected Class<VaultStringCredential> type() {
+        return VaultStringCredential.class;
+    }
+
+    @Override public SingleEnvironment bindSingle(@Nonnull Run<?,?> build,
+                                                  @Nullable FilePath workspace,
+                                                  @Nullable Launcher launcher,
+                                                  @Nonnull TaskListener listener) throws IOException, InterruptedException {
+        return new SingleEnvironment(getCredentials(build).getSecret().getPlainText());
+    }
+
+    @Symbol("string")
```

Comment:
This might conflict with other symbols.
```suggestion
    @Symbol("vaultString")
```